Documentation/CRDs/specification.md

```diff
@@ -1847,6 +1847,18 @@ GatewaySpec
 </tr>
 <tr>
 <td>
+<code>RGWOpsLogEnabled</code><br/>
+<em>
+bool
+</em>
+</td>
+<td>
+<em>(Optional)</em>
+<p>Enable RGW Ops Logs for S3 in a sidecar</p>
```

Comment:
is it only for S3 operations or also swift?

Reply:
@kayrus this should be relevant for swift operations too, but i haven't tested it, you can test that using this setting https://docs.ceph.com/en/latest/radosgw/config-ref/#confval-rgw_enable_ops_log and then exec'ing into the rgw pod and seeing if you get an ops-log file with swift entries